compostables/useFirebase.ts

```diff
@@ -0,0 +1,14 @@
+import { getApps, initializeApp } from "firebase/app";
+import { FirebaseApp } from "@firebase/app-types";
+
+const firebaseConfig = {
+  //Put API keys here
```

Comment:
Should replace it with process.env.FIREBASE_APP_<value> variables. Make an env file locally and make sure we have that in the gitignore before committing (otherwise we'll need to make new credentials)

Actually, we will also need to make check if we need to move these stuff to the server side entirely. Or at least make a call to it. Otherwise these firebase config values might be built into the client side bundle and you could just access them from the browser (and the env stuff wouldn't even matter).